x/axelarnet/client/cli/tx.go

```diff
@@ -172,13 +175,39 @@ func GetCmdRegisterAsset() *cobra.Command {
 		}
 		chain := args[0]
 		denom := args[1]
+		limit := utils.MaxUint
+		window := types.DefaultRateLimitWindow
 
 		isNativeAsset, err := cmd.Flags().GetBool(flagIsNativeAsset)
 		if err != nil {
 			return err
 		}
 
-		msg := types.NewRegisterAssetRequest(cliCtx.GetFromAddress(), chain, nexus.NewAsset(denom, isNativeAsset))
+		limitArg, err := cmd.Flags().GetString(flagLimit)
+		if err != nil {
+			return err
+		}
+		if limitArg != "" {
+			l, ok := sdk.NewIntFromString(limitArg)
+			if !ok || l.IsNegative() {
+				return fmt.Errorf("limit must be a sdk.Uint")
```

Comment:
that means nothing to a user, don't refer to specific data structures (that are not basic types) in error msgs